graphql/types/OrderForm.graphql

```diff
@@ -63,6 +63,12 @@ type OrderFormItem {
   seller: String
   """ Item index on cart """
   cartIndex: Int
+  """ Category ids on orderFormItem """
+  productCategoryIds: String
+  """ Price tags on orderFormItem """
+  priceTags: []
```

Comment:
I think that, in GraphQL schema language, we also need to define the types of properties. Maybe you meant ?
```suggestion
  priceTags: [String]
```